internal/pkg/service/buffer/worker/service/conditions_test.go

```diff
@@ -130,22 +130,22 @@ func TestConditionsChecker(t *testing.T) {
 	// Check conditions checker logs
 	wildcards.Assert(t, `
 %A
-[task][slice.swap/%s]INFO  started task "00000123/my-receiver-B/my-export-2/slice.swap/%s"
-[task][slice.swap/%s]DEBUG  lock acquired "runtime/lock/task/00000123/my-receiver-B/my-export-2/slice.swap/%s"
+[task][slice.swap/%s]INFO  started task "00000123/my-receiver-B/slice.swap/%s"
```

Comment:
Tu by bolo asi fajn zalogovat vacsiu cast toho `lock`, aby bolo nejak overene, ze je zamknuta konkretna slice/file.
Myslim, ze vsetky hodnoty su v tomto teste staticke, ... ak nie tak `%s` staci dat namiesto datumov.

Reply:
Aha, tak tam byla ještě chyba. Tím že ten receiver id a export id už nejsou v prefixu toho zámku, tak se musí explicitně přidat: https://github.com/keboola/keboola-as-code/pull/1183/commits/69208efe459add22784bb9d204dd1e5a647ded16 (Teda jestli to dobře chápu, už v tom začínám mít nějakej zmatek.)

Reply:
jj, to som ti niekde pisal a zabudol skontrolovat, je to tak

Reply:
LGTM